x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/Example.java

```diff
@@ -33,4 +33,6 @@
      * The tag that fences this example.
      */
     String tag();
+
+    String note() default "";
```

Comment:
Added a new field in example annotation, as some functions have notes at the end of the example.